src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -993,8 +993,12 @@ If `P` was bounded, the result is an `HPolytope`; otherwise the result is an
 
 ### Algorithm
 
-- If the unconstrained dimensions of `P` are a subset of the `block` variables,
-  each half-sace `c` of `P` is transformed to `HalfSpace(c.a[block], c.b)`.
+- We first try to exploit the special case where each of the constraints of `P`
```

Comment:
i guess that the necessary `condition constrained_dimensions(P) ⊆ block` is easy to check before doing some work as in lines 1084 - 1090